nixos/modules/programs/winbox.nix

```diff
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg  = config.programs.winbox;
+in
+{
+  options.programs.winbox = {
+    enable = lib.mkEnableOption ("MikroTik Winbox");
+    package = lib.mkPackageOption pkgs "winbox" { };
```

Comment:
This option is unused. Did you forget something?

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/274670